include/aligator/modelling/contact-map.hpp

```diff
@@ -32,25 +36,58 @@ template <typename _Scalar> struct ContactMapTpl {
     if (size_ == 0) {
       ALIGATOR_RUNTIME_ERROR("ContactMap is empty!");
     } else {
+      contact_names_.erase(contact_names_.begin() + long(i));
       contact_states_.erase(contact_states_.begin() + long(i));
       contact_poses_.erase(contact_poses_.begin() + long(i));
       size_ -= 1;
     }
   }
 
+  const std::vector<std::string> &getContactNames() const {
+    return contact_names_;
+  }
+
   const std::vector<bool> &getContactStates() const { return contact_states_; }
 
   bool getContactState(const std::size_t i) const { return contact_states_[i]; }
```

Comment:
Doing getters here is quite abusive I think. There is no need for this, setting these members as public is enough even if God forbid the user decides to misuse and push_back() elements onto them